Makefile

```diff
@@ -32,11 +32,14 @@ format:
 	black -l 100 $(python-packages)
 	isort $(python-packages)
 
-test: check test_python
+test: check test_python_coverage
 
 test_python: $(parser-installed)
 	python3 -m pytest -n$(shell nproc)  -vv --ignore=tests/data tests
 
+test_python_coverage:
+	python3 -m pytest -n$(shell nproc) -vv --cov=librflxlang --cov-branch --cov-fail-under=71 --cov-report=term-missing:skip-covered --ignore=tests/data tests
```

Comment:
Either all `test_python*` targets should have `parser-installed` dependency or none.

Reply:
Removed in d64ef8354f7328cd2ee40b3a77fded360f5c8a98.